data/models/dishes-model.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../dbConfig');
+
+const getDishes = () => {
+  return db('dishes');
+};
+
+const addDish = dish => {
+  return db('dishes').insert(dish);
+};
+
+const getDish = id => {
+  return db('dishes')
+    .leftJoin('recipes', 'dishes.id', 'recipes.dishes_id')
+    .select('dishes.id', 'dishes.name as dish', 'recipes.name as recipe')
+    .where({ 'dishes.id': id });
```

Comment:
this will work, but maybe not with the form that you want--it will return a separate object containing the dish name and recipe for each recipe variant of the same dish, it seems a little repetitive to show the dish name every time. another way of doing this with only one object returned would be to have two separate db calls, where the first is the dish by id, and the second is the recipes by the dish_id as the id--you can make the recipes a property on the dish object so it would be an array of recipes attached to the one dish object